tests/integration/fixtures/mini_sentry.py

```diff
@@ -89,7 +103,7 @@ def store_internal_error_event():
         sentry.test_failures.append(
             (
                 "/api/666/store/",
-                AssertionError("Relay sent us event: {}".format(flask_request.data)),
+                AssertionError("Relay sent us event: %s" % get_error_message(flask_request.data)),
```

Comment:
The error messages reported from sentry were not useful at all and just spammed endless JSON onto stderr. It is totally sufficient to extract the error message and show that in a sanitized way.